themes/default/content/challenge/holiday-shopping.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Holiday Shopping"
+layout: challenge/single
+description: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas 
+meta_desc: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_image: /images/challenge/challenge_cta.png
+---
+
+## Welcome to the Pulumi Challenge!
+
+<div class="flex flex-wrap md:mt-12">
+  <div>
+    <h3>Holiday Shopping List</h3>
+    <p class="pr-12">
+      The holidays are upon us and mere anarchy has been loosed upon our psyche. More specifically how do you keep track of everything you need to buy for gifts or food?!? Surely some salvation is at hand? Lucky for you, our friend, The Pulumipus, has created a simple to-do list application to bring order to your life. In this challenge, you will build this simple to-do list application that has a React frontend, Express backend, and a MongoDB database. This frontend and backend of the app run in Fargate with an Application Load Balancer (ALB) in front and the database is running in a MongoDB Atlas cluster. When you're done, we'll send you a fancy piece of swag, just for this Challenge! If you feel so inclined to thank The Pulumipus, please write a blog post or post a quick video about it. Tag us on social media or email us at <a href=mailto:da@pulumi.com>da@pulumi.com</a>, and we will send you the special swag.
```

Comment:
```suggestion
      The holidays are upon us and mere anarchy has been loosed upon our psyches. More specifically how do you keep track of everything you need to buy for gifts or food?!? Surely some salvation is at hand? Lucky for you, our friend, The Pulumipus, has created a simple to-do list application to bring order to your life. In this challenge, you will build this simple to-do list application that has a React frontend, Express backend, and a MongoDB database. This frontend and backend of the app run in Fargate with an Application Load Balancer (ALB) in front and the database is running in a MongoDB Atlas cluster. When you're done, we'll send you a fancy piece of swag, just for this Challenge! If you feel so inclined to thank The Pulumipus, please write a blog post or post a quick video about it. Tag us on social media or email us at <a href=mailto:da@pulumi.com>da@pulumi.com</a>, and we will send you the special swag.
```

Reply:
>When you're done, we'll send you a fancy piece of swag
>Tag us on social media or email us and we will send you the special swag

Are there two pieces of swag, then (one you generate yourself and an extra one for tagging or emailing? We should probably make this clear up front so there isn't any confusion about it.